docs/src/app/components/pages/components/Card/ExampleControlled.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Card from 'material-ui/lib/card/card';
+import CardActions from 'material-ui/lib/card/card-actions';
+import CardHeader from 'material-ui/lib/card/card-header';
+import CardMedia from 'material-ui/lib/card/card-media';
+import CardTitle from 'material-ui/lib/card/card-title';
+import FlatButton from 'material-ui/lib/flat-button';
+import CardText from 'material-ui/lib/card/card-text';
+import Toggle from 'material-ui/lib/toggle';
+
+export default class CardExampleControlled extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      expanded: false,
+    };
+  }
+
+  handleExpandChange = (expanded) => {
+    this.setState({expanded: expanded});
+  };
+
+  handleToggle = (event, toggle) => {
+    this.setState({expanded: toggle});
+  };
+
+  handleExpand = () => {
+    this.setState({expanded: true});
+  };
+
+  handleReduce = () => {
+    this.setState({expanded: false});
+  };
+
+  render() {
+    return (
+      <Card expanded={this.state.expanded} onExpandChange={this.handleExpandChange}>
+        <CardHeader
+          title="URL Avatar"
+          subtitle="Subtitle"
+          avatar="http://lorempixel.com/100/100/nature/"
+          actAsExpander={true}
+          showExpandableButton={true}
+        />
+        <CardText>
+          This toggle control the expanded state of the component.
```

Comment:
If you think I'm making a PR for a one letter change you have another thing coming. :stuck_out_tongue_winking_eye: